### DISCUSSION TITLE
How can i do a real time listener with prisma?

### DISCUSSION THREAD

**Discussion by j5gomes** [prisma/prisma#8405] [Q&A] [answered]
Hello, How can i do a real time listener with prisma?

my code:

```ts
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()

async function save() {
  await prisma.user.create({
    data :{
      name: 'JoJo',
      email: 'jojo@joaogomes.dev',
      Post: {
        create: [
          {title: 'Post 1', content: 'Hello'},
          {title: 'Post 2', content: 'World'}
        ]
      }
    }
  })

  console.log("Saved");
}

async function list() {
  const allUsers = await prisma.user.findMany({
    include: {
      Post: true
    }
  });
  console.log(allUsers);
}

save()
  .catch((e) => {
    throw e
  })
  .finally(async () => {
    await prisma.$disconnect()
  })

list()
.catch((e) => {
  throw e
})
.finally(async () => {
  await prisma.$disconnect()
})
```
Thanks!!

**Comment by ryands17** [comment, marked as answer]
@JoaoGomes5 👋 

Prisma doesn't have any built-in realtime functionality so you would either need to use native WebSockets or a hosted solution like [Supabase](https://supabase.io/).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
